File: java1/my_http_ex/src/ch01/Photo.java
package ch01;

public class Photo {
	
	int albumId;
	int id;
	String title;
	String url;
	String thumbnailUrl;
	@Override
	public String toString() {
		return "Photo [albumId=" + albumId + ", id=" + id + ", title=" + title + ", url=" + url + ", thumbnailUrl="
				+ thumbnailUrl + "]";
	}
	
	

}

File: flutter_work/basic_http_flutter_3/lib/models/photo.dart
class Photo {
  var albumId;
  var id;
  var title;
  var url;
  var thumbnailUrl;

  Photo({this.albumId, this.id, this.title, this.url, this.thumbnailUrl});

  Photo.init(this.title);

  Photo.formJson(Map<String, dynamic> photoMapData)
      : albumId = photoMapData["albumId"],
        id = photoMapData["id"],
        title = photoMapData["title"],
        url = photoMapData["url"],
        thumbnailUrl = photoMapData["thumbnailUrl"];

  @override
  String toString() {
    return 'Photo{albumId: $albumId, id: $id, title: $title, url: $url, thumbnailUrl: $thumbnailUrl}';
  }
}
